languages/swift/exercises/concept/enums/.meta/Example.swift

```diff
@@ -0,0 +1,29 @@
+enum LogLevel: Int {
+  case trace
+  case debug
+  case info = 4
+  case warning
+  case error
+  case fatal
+  case unknown = 42
+
+  init(_ line: String) {
+    guard let lead = line.first, lead == "[" else {
+      self = .unknown
+      return
+    }
+    switch line.dropFirst().prefix(3) {
```

Comment:
Are students familiar with `switch` statements at this point? If not, you might want to add it as a prerequisite

Reply:
Yes. They were introduced to them a couple of exercises back, in `conditionals`.

Reply:
👍